src/components/LocationPage/LocationPageHeader.tsx

```diff
@@ -47,6 +48,12 @@ const NewFeatureCopy = (props: {
   );
 };
 
+const HospitalizationsAlert: React.FC<{ projection: Projection }> = ({
+  projection,
+}) => {
+  return <Copy>{getHospitalizationsAlert(projection)}</Copy>;
```

Comment:
Not really sure what the point of this component is if it basically just defers to `getHospitalizationsAlert()`.  I feel like this could be refactored a bit (probably to pull the logic from getHospitalizationsAlert() into this component?).